src/telemetry/contextShapes.ts

```diff
@@ -108,7 +108,7 @@ export const contextShapes = (
         apiData.namespace = namespace;
       }
 
-      const body = encode("ContextShapes", apiData);
```

Comment:
My guess is you need to see a specific "ContextShapes" key in this body object vs. just the apiData object?
```ts
body: JSON.stringify({ContextShapes: apiData}),
```